recipes/mavsdk/all/conanfile.py

```diff
@@ -0,0 +1,167 @@
+from os.path import join
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.files import replace_in_file, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.scm import Git
+from conan.tools.env import Environment
+
+required_conan_version = ">=1.53.0"
+
+
+class MAVSDKConan(ConanFile):
+    name = "mavsdk"
+    license = "BSD-3-Clause"
+    author = "SINTEF Ocean"
+    homepage = "https://mavsdk.mavlink.io/main/en/"
+    url = "https://github.com/mavlink/MAVSDK.git"
+    description = "C++ library to interface with MAVLink systems"
+    settings = "os", "compiler", "build_type", "arch"
+    package_type = "library"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False]
+        }
+    default_options = {
+        "fPIC": True,
+        "shared": False
+        }
+
+    @property
+    def export_sources(self):
+        pass
+        #export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
```suggestion
    implements = ["auto_shared_fpic"]
```

will take care of this, and the missing `configure()` method that handles fpic values when building shared libraries